composer.json

```diff
@@ -18,35 +29,38 @@
             "name": "Henrik Bjornskov"
         }
     ],
-    "support": {
-        "issues": "https://github.com/friends-of-phpspec/phpspec-code-coverage/issues",
-        "source": "https://github.com/friends-of-phpspec/phpspec-code-coverage",
-        "docs": "https://github.com/friends-of-phpspec/phpspec-code-coverage#phpspec-code-coverage"
-    },
     "require": {
         "php": "^7.1",
-        "phpspec/phpspec": "^4.2||^5.0",
-        "phpunit/php-code-coverage": "^5.0||^6.0||^7.0"
+        "phpspec/phpspec": "^4.2 || ^5.0",
+        "phpunit/php-code-coverage": "^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "escapestudios/symfony2-coding-standard": "^3.1",
-        "squizlabs/php_codesniffer": "^3.2"
+        "drupol/php-conventions": "^1.6",
```

Comment:
Hi, it's basically a list of custom rules, inherited from PSR12 and Symfony rules.

Reply:
Yes I understand but it's a personal shortcut. We are creating this new org / project to avoid being tied too much with a single person. I think that we need to adopt more global rules like PSR1-2 only here.

Reply:
There's nothing fancy, you can see the rules here: https://github.com/drupol/phpcsfixer-configs-php/tree/master/config

Rules are inherited:

* PSR12 > PHP > PHP56
* PHP56 > PHP7
* PHP7 > PHP71
* PHP71 > PHP72
* PHP72 > PHP73

PHP CS Fixer Configuration objects are in: https://github.com/drupol/phpcsfixer-configs-php/tree/master/src/Config

Reply:
PSR2 has been deprecated, why not use PSR12 only then ?

Reply:
Totally agree here. I understand that there is nothing fancy, but, personaly, when I check a project and find that there are custom rules applied on it (here it's a combination of PSR12 + Symfony rules) and some other tools to rely on (GrumPHP) which are layers over standard ones I'm not really confident. It seems to rely too much on specific persons.

I don't mean that your conventions are not useful but also, there is only a small amount of PHP code here and I really don't want to complicate usage for a so small library.

In such case, I'll go for a basic combination of phpcs + PSR1/12 and that's all.

@jaylinski what's your POV here ?

Reply:
I just pushed a commit https://github.com/friends-of-phpspec/phpspec-code-coverage/pull/10/commits/f0ebde7ee20e0aae1866d896840ed1acbd55ffe2 that show the difference between the coding standard that I used here and PSR1/PSR12.

To be honest, I would not use only PSR1/PSR12, see the differences by yourselves and let me know.

Reply:
See it in action:

![image](https://user-images.githubusercontent.com/252042/65952131-b27c2b80-e441-11e9-9d75-2af91647cb89.png)

Reply:
Those tasks are defined in this file: https://github.com/drupol/php-conventions/blob/master/config/psr12/grumphp.yml